src/javascript/upgrade-dependencies.ts

```diff
@@ -388,6 +391,21 @@ export class UpgradeDependencies extends Component {
     return dependencies.map((d) => d.name);
   }
 
+  /**
+   * Projen only uses npm-check-updates to upgrade dependencies when either the version is omitted, or set to "*".
+   * Otherwise, the exact version selected is placed in the package.json file and upgrading is handled through the package manager
+   * rather than npm-check-updates.
+   *
+   * @param version semver from DependencyCoordinates.version, may be undefined
+   * @returns whether the version is the default versioning behavior
+   */
+  private packageIsUsingDefaultVersioning(
+    version: string | undefined
+  ): boolean {
+    // No version means "latest"
+    return !version || version === "*";
+  }
```

Comment:
Let's give this a more descriptive name
```suggestion
  private isProjenManagedVersion(
    version: string | undefined
  ): boolean {
    // No version means "latest"
    return version && version !== "*";
  }
```

Reply:
I think the name doesn't reverse the logic here... "Projen Managed" would be anything where a version is not provided, or IS star...

I'll change the name, but keep the logic.